### DISCUSSION TITLE
Function to create Bluesky Scenarios

### DISCUSSION THREAD

**Discussion by turrengoechea** [TUDelft-CNS-ATM/bluesky#479] [Q&A]
Hi, I want to make a function to create scenarios in Bluesky with python. How can i do it?

**Comment by ProfHoekstra** [comment]
Scenario files are simple text files with the extension .SCN. So you can write them in Python using print:

f = open(“myscensrio.scn”,”w”)
print (……,file=f)
f.close()

Check out the example scenario files. The format is for each line: a time stamp (starting at zero for the start of the run) and the carrot “>” followed by any command as you would enter it in the command window:

HH:MM:SS.hh> command line

E.g.:

00:00:21.04>KL204 ALT 2000

See the command reference in the wiki for commands like CRE and ADDWPT, DEST etc to create your scenarios.

Best regards,
    Jacco Hoekstra


On 3 Sep 2023, at 11:01, Tomás De Urrengeochea Cantavenera ***@***.***> wrote:

﻿

Hi, I want to make a function to create scenarios in Bluesky with python. How can i do it?

**Comment by turrengoechea** [comment]
Thank you for your replay
